fix layout complete

DamianJorga opened the pull request.

DamianJorga committed:
fix layout complete

greg-top commented on index.html:
```diff
@@ -10,8 +10,119 @@
+            </div>
+        </div>
+        <div class="footer-bottom row">
+            <div class="col-mobile-100">
```
Hey, z tego co pamietam to footer-bottom mial 3 kolumny na desktop ;)

greg-top commented on index.html:
```diff
@@ -10,8 +10,119 @@
+        </div>
+    </main>
 
+    <footer class="container-fluid">
```
![image](https://user-images.githubusercontent.com/38762883/84188599-c590d200-aa8b-11ea-80f9-6a0d2cd76cb0.png)
Tutaj bd potrzebna zmiana bo bd mial problem z ostylowaniem kolumn footera. Ja bym to zmienil tak:
1.) Usunal byl klase container-fluid ze znacznika footer
2.) Wewnatrz znacznika footer utworzylbym 2 divy dla:
    - newsletter (ten powinien byc container-fluid)
    - footer-bottom (ten powinien byc container)
Dzieki temu latwiej ci bedzie ostylowac footer bottom (zauwaz ze jego kolumny nie sa fluid)

greg-top commented on index.html:
```diff
@@ -10,8 +10,119 @@
+        </div>
+        <!-- TODO: fix layout -->
+        <div class="three-columns shop row">
+            <div class="col-mobile-100 col-large-tablet-33">
```
hey,
![image](https://user-images.githubusercontent.com/38762883/84188976-6089ac00-aa8c-11ea-9e47-c8403d5ca5be.png)

tutaj brakuje kolumny dla dolnego siodelka, dodales kolumny 50*2 dla 2 malych zdjec, ale zapomniales dodac kolumne 100 dla dolnego zdjecia (ona sobie splynie do kolejnej linii)

greg-top commented on index.html:
```diff
@@ -10,8 +10,119 @@
+                <div class="col-large-tablet-100">
+                    <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Iusto minus sunt, soluta facere, quidem deleniti possimus repellat ab id doloremque illum voluptas? Nemo, nulla. Laborum doloribus earum maxime. Voluptatum, ipsum!</p>
+                </div>
+                <div class="col-large-tablet-100">
```
![image](https://user-images.githubusercontent.com/38762883/84189196-cb3ae780-aa8c-11ea-8e33-d65d3ec3303f.png)
tutaj na dole sa 2 odrebne zdjecia, wiec nie mozesz uzyc kolumny 100, powinno byc 2x po 50

greg-top commented on index.html:
```diff
@@ -10,8 +10,119 @@
+        <!-- TODO: fix layout -->
+        <div class="three-columns shop row">
+            <div class="col-mobile-100 col-large-tablet-33">
+                <div class="col-large-tablet-50">
```
w calym kontenerze musisz rowniez dodac klase col-mobile-100, inaczej mozesz miec bledy dla mobile, powtorz to we wszystkich kolumnach tego kontenera

greg-top reviewed (changes requested):
Hey Damian, komentarze bd pisali po polsku, prosze abys pisal nazwy commitow oraz branchy dla git po angielsku ;)

DamianJorga committed:
improved footer and fix layout and graphics for shop now

DamianJorga commented:
I added 2 columns to the footer-bottom, footer tag and corrected content. last I don't know if I understand correctly I added col-mobile-100 to everything

DamianJorga committed:
add class col-mobile-100 line 82

greg-top commented:
Wszystko gra teraz :)

greg-top reviewed (approved):
ok :)
